test/grant.mfaOtp.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright 2013-present NightWorld.
```

Comment:
They didn't write this code, so let's at least not put their copyright in this file :)

Reply:
lol! Fixed, thanks!